app/configurations/config.varely.js

```diff
@@ -170,4 +170,5 @@ export default configMerger(walttiConfig, {
   showNearYouButtons: true,
   allowLogin: false,
   routeNotifications: [],
+  analyticsScript: '',
```

Comment:
maybe not needed? Empty string vs function in waltti config.

Reply:
The config is needed here to stop the analytics script from being added to varely instance from waltti config. Empty string is the minimalist way to stop it but I was wondering if it's a little bit confusing.

Reply:
Would a void function `() => {}` be less confusing?

Reply:
Empty string works as javascript thinks it is a 'falsy' value.  I guess we can accept this.